.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,10 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Ideally these files are just all deleted and ignored. But if you want me to manually exclude these changes please let me know